### PR TITLE
Update pkgdown URL

### PR DESCRIPTION
**Pull request by maelle**
pgkdown docs are built centrally https://devdevguide.netlify.app/maintenance_cheatsheet.html#pkgdown-documentation :-) (so you should also remove the pkgdown workflow, maybe only adding a workflow that runs `pkgdown::check_pkgdown()` example https://github.com/ropensci/webchem/blob/master/.github/workflows/check-pkgdown.yml )

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-url: https://ropensci.github.io/openalexR/
-template:
-  bootstrap: 5
-  bslib:
-    primary: "#BB4827"
-    border-radius: 0.5rem
-    btn-border-radius: 0.25rem
+url: https://docs.ropensci.org/openalexR/
+
+authors:
```
Comment (maelle) [inline _pkgdown.yml:3]: this means that on the sidebar of the pkgdown website your names will be linked to your websites https://pkgdown.r-lib.org/reference/build_home.html#yaml-config-authors

**Comment by maelle** [comment]
I forgot! If you remove the current pkgdown website, refer to https://devdevguide.netlify.app/redirect.html to set up a redirect.

**Review by trangdata — APPROVED** [review]
Thanks so much for the changes and for helping me with the redirect. I assume index.html and 404.html mentioned in the [devguide](https://devguide.ropensci.org/redirect.html) would need to be added to the gh-pages branch?

Please feel free to merge this!

**Comment by maelle** [comment]
> Thanks so much for the changes and for helping me with the redirect. I assume index.html and 404.html mentioned in the [devguide](https://devguide.ropensci.org/redirect.html) would need to be added to the gh-pages branch?

Yes!

> Please feel free to merge this!

I'll let you merge it! (especially as I just fixed conflict, I prefer you to check I did it correctly :grin: )

**Pull request merged**